common/api/maindb.py

```diff
@@ -159,7 +159,7 @@ def init_maindb():
 
 
 
-@transaction.commit_manually
+# @transaction.commit_manually
```

Comment:
Fixed in e85f995e4048f10a930e653a753ee5b7e1b46752 which also fixed the module-level docstring.